src/data/invalid/DataObject-in_manifest-invalid.yaml

```diff
@@ -0,0 +1,11 @@
+#this test should fail because the value of in_manifest should be a list, not a list of lists
```

Comment:
The JSON equivalent of the `in_manifest` value below is:

```json
  "in_manifest": [
    "['nmdc:manif-11-7796sg87']"
  ]
```

It is already a list of strings (as opposed to a list of lists)—but the string, itself, does not have the structure I think schema authors expect.